docs/user/service-catalog.md

```diff
@@ -0,0 +1,275 @@
+externalDatabase.database: database value in the binding secret
+mariadb.enabled: false (to avoid deploying a MariaDB database)
+```
+
+Once those values have been modified, click on `Submit` and wait until the
+deployment is completed:
+
+![Wordpress deployed](../img/wordpress-deployed.png)
+
+If we check the wordpress pod log we can see that it connected successfully
+to the Azure MySQL database:
+
+```
+k logs wordpress-app-wordpress-597b9dbb5-2rk4k
+
+Welcome to the Bitnami wordpress container
+Subscribe to project updates by watching https://github.com/bitnami/bitnami-docker-wordpress
+Submit issues and feature requests at https://github.com/bitnami/bitnami-docker-wordpress/issues
+
+WARN  ==> You set the environment variable ALLOW_EMPTY_PASSWORD=yes. For safety reasons, do not use this flag in a production environment.
+nami    INFO  Initializing apache
+apache  INFO  ==> Patching httpoxy...
+apache  INFO  ==> Configuring dummy certificates...
+nami    INFO  apache successfully initialized
+nami    INFO  Initializing php
+nami    INFO  php successfully initialized
+nami    INFO  Initializing mysql-client
+nami    INFO  mysql-client successfully initialized
+nami    INFO  Initializing libphp
+nami    INFO  libphp successfully initialized
+nami    INFO  Initializing wordpress
+mysql-c INFO  Trying to connect to MySQL server
+mysql-c INFO  Found MySQL server listening at 93489418-7a54-4b2c-b807-d239cb26ad5d.mysql.database.azure.com:3306
+mysql-c INFO  MySQL server listening and working at 93489418-7a54-4b2c-b807-d239cb26ad5d.mysql.database.azure.com:3306
+```
+
+## Example 2. Chartmuseum using a GCP storage bucket
+
+[ChartMuseum](https://github.com/helm/chartmuseum) is an open-source project
+that allows you to easily have your own valid Helm Chart Repository. It has
+support for different cloud storage backends, including [Google Cloud Storage](https://cloud.google.com/storage/).
+
+In this example we will create a Google Cloud Storage instance with Kubeapps
+Service Catalog integration, and we will use it as backend for a ChartMuseum
+deployment.
+
+### Provisioning the Google Cloud Storage bucket.
+
+Under `Service Instances` click on `Deploy Service Instance` and select Google
+Cloud Storage:
+
+![Google Cloud Storage](../img/gcp-storage.png)
+
+Select the `Beta plan` and provide a name. 
+
+We will be adding some parameters to configure the name of the bucket and
+the location. We will leave the rest of the options with the default values:  
+
+![Google Cloud Storage options](../img/gcp-storage-options.png)
+
+Once provisioned, we will add a binding clicking on `Add binding`:
+
+![Google Cloud Storage binding options](../img/gcp-storage-binding-options.png)
+
+That will create a new service account in your GCP project with the right
+persmissions to access the bucket.
+
+Once the binding creation has been completed, the details of the created
+secret can be explored directly from the same page, clicking on `show`.
```

Comment:
Again, we should probably tell the user to note down the credentials for the next step.